src/components/Sidebar/Sidebar.tsx

```diff
@@ -18,32 +47,187 @@ const Sidebar = () => {
+            </SidebarSectionItem>
+          </SidebarSectionBody>
+        </SidebarSection>
+
+        <SidebarSection>
+          <SidebarSectionHeader>Payments</SidebarSectionHeader>
+          <SidebarSectionBody>
+            <SidebarSectionItem href="/payments/transfers">
+              <ArrowUpTrayIcon className={iconsClasses} />
+              Transfers
+            </SidebarSectionItem>
+            <SidebarSectionItem href="/payments/history">
+              <ArchiveBoxIcon className={iconsClasses} />
+              History
+            </SidebarSectionItem>
+            <SidebarSectionItem href="/beneficiary/manage">
+              <BookOpenIcon className={iconsClasses} />
+              Address Book
+            </SidebarSectionItem>
+          </SidebarSectionBody>
+        </SidebarSection>
+
+        <SidebarSection>
+          <SidebarSectionHeader>Approvals</SidebarSectionHeader>
+          <SidebarSectionBody>
+            <SidebarSectionItem href="/approval/pending">
+              <QueueListIcon className={iconsClasses} /> Pending requests
+            </SidebarSectionItem>
+            <SidebarSectionItem href="/approval/manage">
+              <AdjustmentsVerticalIcon className={iconsClasses} /> Manage
+              wallets
+            </SidebarSectionItem>
+            <SidebarSectionItem href="/approval/create">
+              <PlusCircleIcon className={iconsClasses} /> Create multisig
+              wallets
+            </SidebarSectionItem>
+          </SidebarSectionBody>
+        </SidebarSection>
+
+        <SidebarSection>
+          <SidebarSectionHeader>Accounting</SidebarSectionHeader>
+          <SidebarSectionBody>
+            <SidebarSectionItem href={"/accounting/tta"}>
+              <PieChartIcon className={iconsClasses} /> Reports
+            </SidebarSectionItem>
+          </SidebarSectionBody>
+        </SidebarSection>
       </div>
+
+      {selectedAccount ? (
+        <SelectAccount openWalletModal={openWalletModal} />
+      ) : (
+        <Button onClick={openWalletModal}>Connect Account</Button>
+      )}
+
       <WalletModal />
-    </>
+    </div>
+  );
+}
+
+function SidebarSection({ children }: { children: React.ReactNode }) {
+  return <div className="px-3 py-2">{children}</div>;
+}
+
+function SidebarSectionHeader({ children }: { children: React.ReactNode }) {
+  return (
+    <h2 className="mb-2 px-4 text-lg font-semibold tracking-tight">
+      {children}
+    </h2>
+  );
+}
+
+function SidebarSectionBody({ children }: { children: React.ReactNode }) {
+  return <div className="space-y-1">{children}</div>;
+}
+
+function SidebarSectionItem({
+  children,
+  className,
+  href,
+}: {
+  children: React.ReactNode;
+  className?: string;
+  href: string;
+}) {
+  return (
+    <Link href={href}>
+      <Button variant="ghost" className={cn("w-full justify-start", className)}>
+        {children}
+      </Button>
+    </Link>
   );
-};
+}
+
+function TeamsDropdownMenu() {
```

Comment:
Would be nice to have 1 component per file TeamsDropdownMenu -> TeamsDropdownMenu.tsx